types/chroma-js/package.json

```diff
@@ -21,5 +21,15 @@
             "name": "Charlie Zhuo",
             "githubUsername": "CharlieZhuo"
         }
-    ]
+    ],
+    "exports": {
+        ".": {
+            "import": "./index.d.mts",
+            "require": "./index.d.ts"
+        },
+        "./light": {
+            "import": "./index.d.mts",
+            "require": "./index.d.ts"
+        }
+    }
```

Comment:
FWIW, typically we expect the exports to match the upstream. In this case, the upstream has a bunch more, and are documented: https://gka.github.io/chroma.js/#:~:text=If%20you%20just%20want%20to%20use%20parts%20of%20chroma.js%20and%20not%20bundle%20the%20entire%20package%2C%20you%20can%20import%20directly%20from%20chroma%2Djs/src/*%20to%20benefit%20from%20treeshaking.

It's possible that a more complete PR should actually model the file structure of the real package, though it's probably "okay" for now in this PR?